node-backend/app/controllers/v1/organisationController.js

```diff
@@ -1,10 +1,11 @@
 const yup = require("yup");
 
+const utility = require("../../utils/utility");
 const db = require("../../models/sequelize");
 const Organizations = db.organizations;
 
 const schema = yup.object().shape({
```

Comment:
define all schema objects required in another file and include it here for validation. So you will have `validationSchema` folder inside ` controllers/v1` folder. In that for organisation you will have `orgValidationSchema.js`. In this file you will define various validation schemas like insertSchema, getSchema etc. And you will use them in your controller to validate. This will make code more readable